identify/README.md

```diff
@@ -133,3 +134,10 @@ clients only support initiating requests while some servers (only) support
 responding to requests. To prevent clients from initiating requests to other
 clients, which given them being clients they fail to respond, clients should not
 advertise `foo` in their `protocols` list.
+
+### signedPeerRecord
+
+This is a serialized [SignedEnvelope](https://github.com/libp2p/go-libp2p/blob/master/core/record/pb/envelope.proto) containing a [PeerRecord](https://github.com/libp2p/go-libp2p/blob/master/core/peer/pb/peer_record.proto),
```

Comment:
Can this link to specs rather than implementations?  If it must link to implementations, it should use the [permalink](https://docs.github.com/en/repositories/working-with-files/using-files/getting-permanent-links-to-files) format that has the commit hash embedded so future refactorings don't break links

e.g:

```
https://github.com/libp2p/go-libp2p/blob/d55bed5f78fad923353b92c7d1db64c758bcbf9c/core/record/pb/envelope.proto
```